src/main/java/Lesson3/Task2/Plane.java

```diff
@@ -0,0 +1,17 @@
+package Lesson3.Task2;
+
+class Plane extends Vehicle {
+    int height;
+    int passPlaces;
+    int year;
+
+    Plane(int height, int passPlaces, int year) {
+        this.height = height;
+        this.passPlaces = passPlaces;
+        this.year = year;
```

Comment:
Here you need to pass all common parameters(coordinates, price, speed, year) + Plane's own parameters(height, passPlaces)